cg/meta/workflow/utils/are_all_samples_control.py

```diff
@@ -0,0 +1,11 @@
+from cg.constants.constants import ControlOptions
```

Comment:
Is there a reason for us creating a whole file for this utils method rather than putting it in a utils.py file?

Reply:
In which utils.py file should I put it?

Reply:
The placement is somewhat fine. I just have not seen them being split into separate files so I would have probably put it in a /cg/meta/workflows/utils.py file. Not a super strong preference though, so feel free to go rogue.

Reply:
I agree that it looks a bit weid to have one utils module with one function, and it is not scalable. What I see is that there are two util modules in this directory holding only one function, maybe they could be merged into one module simply called utils. So instead of:
```shell
workflow/
├── utils/
│   ├── get_genome_build.py
│   └── are_all_samples_control.py
├── __init____.py
├── analysis.py
├── balsamic.py
├── ...
└── tomte.py
```
have
```shell
workflow/
├── __init____.py
├── analysis.py
├── balsamic.py
├── ...
├── utils.py
└── tomte.py
```
where `utils.py` holds the two functions

This could be out of scope for this PR, so I think @eliottBo can reserve the right to resolve this conversation and implement this in another PR.

Reply:
I will rename the file I created utils.py and future utils functions can be added